queries.yaml

```diff
@@ -137,6 +138,11 @@ sources:
     - template: wof.jinja2
       start_zoom: 11
 
+  admin_areas:
+    - template: admin_areas.jinja2
+      # starts at the min zoom where we have OSM roads
```

Comment:
`# starts at the min zoom where we have roads layer content`

Right now it's NE roads at 5+, and switches to OSM later at 9.